test/unit/src/common/reducers/entities/t_entities.js

```diff
@@ -3,21 +3,33 @@ import proxyquire from 'proxyquire';
 
 import * as fixtures from './fixtures.js';
 import * as RepoActionTypes from '../../../../../../src/common/actions/repository';
+import * as SnapsActionTypes from '../../../../../../src/common/actions/snaps';
 import * as RegisterNameActionTypes from '../../../../../../src/common/actions/register-name';
 
-const repoSpy = expect.createSpy();
-const snapSpy = expect.createSpy();
+describe('entities reducer', function() {
+  let repoSpy;
+  let snapSpy;
 
-const entities = proxyquire('../../../../../../src/common/reducers/entities', {
-  './repository': {
-    default: repoSpy
-  },
-  './snap': {
-    default: snapSpy
-  }
-}).entities;
+  let entities;
+
+  beforeEach(() => {
+    repoSpy = expect.createSpy();
+    snapSpy = expect.createSpy();
+
+    entities = proxyquire('../../../../../../src/common/reducers/entities', {
+      './repository': {
+        default: repoSpy
+      },
+      './snap': {
+        default: snapSpy
+      }
+    }).entities;
+  });
+
+  afterEach(() => {
+    expect.restoreSpies();
```

Comment:
The spies were not properly cleared before, so some of the results have been false positives.

Should be fixed now.